packages/apps-engine/src/definition/livechat/ILivechatRoom.ts

```diff
@@ -21,6 +21,8 @@ interface IOmnichannelSourceApp {
     label?: string;
     sidebarIcon?: string;
     defaultIcon?: string;
+    // The destination of the message (e.g widget host, email address, whatsapp number, etc)
+    destination?: string;
```

Comment:
Don't we have this data already? The `label` prop for Whatsapp channels holds the phone number it was directed to. The email address is on an `email` prop, sms number is on an `sms` prop on the room. Any reason for adding another prop?

Reply:
It looks like you're not using this destination for anything yet, right? 
In that case we can leave it out of this PR and implement it separately if it adds value.

Reply:
We are, it is saving some information in the `livechat/room` endpoint, on the apps.engine bridge and also on the sms endpoint. Also this make available for the apps to save the info in a appropriated field

https://github.com/RocketChat/Rocket.Chat/pull/33308/files#diff-9e26aa53a3f9ad45396b8eb88faf7ee7ca9629d0c60544f27aadae68b2ec4017R80
https://github.com/RocketChat/Rocket.Chat/pull/33308/files#diff-5174fa8b7d018b6625c680bcf2e0ef6b319a9577b4856aca8df8073d508263d2R134
https://github.com/RocketChat/Rocket.Chat/pull/33308/files#diff-5b21e788e68edd23d55d2b1ba9c7610bd20a06ae63a9689bad84625c153604cfR121